numpy/distutils/mingw32ccompiler.py

```diff
@@ -85,17 +85,28 @@ def __init__ (self,
         elif self.linker_dll == 'gcc':
             self.linker = 'g++'
 
-        # **changes: eric jones 4/11/01
-        # 1. Check for import library on Windows.  Build if it doesn't exist.
+        p = subprocess.Popen(['gcc', '--version'], shell=True,
+                             stdout=subprocess.PIPE)
```

Comment:
don't we have this somewhere else already? using -dumpversion as its easier to parse